pkg/endpoint/kubernetes/cep.go

```diff
@@ -0,0 +1,367 @@
+// Copyright 2016-2018 Authors of Cilium
```

Comment:
what if we move `pkg/endpoint/kubernetes` to `pkg/k8s/cilium_endpoint` instead?

Reply:
I've done this. The only drawback here is that this puts the endpoint dependency into `pkg/k8s`.